frontend/src/App.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: 'numbers';
+  src: url('./numbers.ttf')
+}
+
+body {
+  font-family: 'numbers'; 
```

Comment:
co to za family to numbers?

Reply:
poprawione, teraz sie nazywa "specialFont"